menas/ui/css/style.css

```diff
@@ -127,3 +127,7 @@ h5[id$=datasetDetailView--scheduleTimingTitle] {
   padding: 8px 2px;
   margin-top: 4px;
 }
+
+.italic{
```

Comment:
I would not name the class _how it is_, but _what it is_. E.g. `missingPropertyValue`

Reply:
There is already the `italicMissingProp` formatter, so I would consider leaving that class `italic` in case it would be reused for other purposes

Reply:
That's the thing. I am not sure it should be reused as it is. Such a class is not different from assigning a style directly. Not very modular.